### ISSUE TITLE
Cannot read property 'iterator' of undefined

### ISSUE THREAD

**Issue by pfrazee** [mafintosh/hypercore#82]
Got hit with this error in some tests, couldn't reproduce with a re-run but I figured I'd file an issue for now.

```
Error: Cannot read property 'iterator' of undefined

  Peer.update (node_modules/hypercore/lib/replicate.js:257:13)
  Feed._updatePeers (node_modules/hypercore/index.js:675:61)
  Feed.download (node_modules/hypercore/index.js:240:8)
  Promise (lib/network.js:41:25)
  onFulfilled (node_modules/co/index.js:65:19)
```

**Comment by pfrazee** [comment]
Got this again:

```
Uncaught Exception:
TypeError: Cannot read property 'iterator' of undefined
    at Peer.update (/Users/paulfrazee/my/hyperdrive/node_modules/hypercore/lib/replicate.js:257:13)
    at Feed._updatePeers (/Users/paulfrazee/my/hyperdrive/node_modules/hypercore/index.js:718:61)
    at Peer._clear (/Users/paulfrazee/my/hyperdrive/node_modules/hypercore/lib/replicate.js:79:13)
    at /Users/paulfrazee/my/hyperdrive/node_modules/hypercore/lib/replicate.js:59:10
    at done (/Users/paulfrazee/my/hyperdrive/node_modules/last-one-wins/index.js:40:5)
    at ondone (/Users/paulfrazee/my/hyperdrive/node_modules/hypercore/index.js:906:5)
    at onwrite (/Users/paulfrazee/my/hyperdrive/node_modules/random-access-file/index.js:143:25)
    at FSReqWrap.wrapper [as oncomplete] (fs.js:672:5)
```

**Comment by mafintosh** [comment]
Should be fixed in 5.11.1. If not please reopen

**Comment by pfrazee** [comment]
Thanks!
